tss-esapi/tests/Dockerfile-ubuntu

```diff
@@ -7,10 +7,11 @@ ENV PATH="/root/.cargo/bin:${PATH}"
 
 FROM rust-toolchain AS tpm2-tss
 # Download and install the TSS library
-ENV TPM2_TSS_BINDINGS_VERSION=3.2.2
+ENV TPM2_TSS_BINDINGS_VERSION=4.0.1
 ARG TPM2_TSS_VERSION=$TPM2_TSS_BINDINGS_VERSION
 ENV TPM2_TSS_VERSION=$TPM2_TSS_VERSION
-ENV PKG_CONFIG_PATH /usr/local/lib/pkgconfig
+ENV PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
```

Comment:
Why remove the equals sign here while it's there a couple of lines above? :thinking:

Reply:
I think I got a warning when I was building the docker about deprecated initialization of variables or something like that. And that is why I changed it.